### PR TITLE
Create observation config for ZTF

### PR DESCRIPTION
**Pull request by rmorgan10**
I have tried to mirror the other ObservationConfig files to the best of my ability, all sources of ZTF information are cited in the comments.

### CODE REVIEW

File: lenstronomy/SimulationAPI/ObservationConfig/ZTF.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+:keyword sky_brightness: sky brightness (in magnitude per square arcseconds in units of electrons)
+:keyword magnitude_zero_point: magnitude in which 1 count (e-) per second per arcsecond square is registered
+:keyword num_exposures: number of exposures that are combined (depends on coadd_years)  
+:keyword seeing: Full-Width-at-Half-Maximum (FWHM) of PSF
+:keyword psf_type: string, type of PSF ('GAUSSIAN' supported) 
+"""
+
+class ZTF(object):
+    """
+    class contains DES instrument and observation configurations
```
Comment (sibirrer) [inline lenstronomy/SimulationAPI/ObservationConfig/ZTF.py:59]: maybe change the description  :)

**Review by sibirrer — CHANGES_REQUESTED** [review]
thanks for adding ZTF! Can you please add a test to make sure these keyword arguments can make an instance of the Instrument() class? See other observational config tests. Thanks

**Comment by sibirrer** [comment]
ah, and you can add yourself to the AUTHORS.rst file with this PR too!

File: lenstronomy/SimulationAPI/ObservationConfig/ZTF.py (hunk 1 above)
Comment (rmorgan10) [reply lenstronomy/SimulationAPI/ObservationConfig/ZTF.py:59]: Whoops haha

Comment (rmorgan10) [reply lenstronomy/SimulationAPI/ObservationConfig/ZTF.py]: Test added and description fixed

**Review by sibirrer — APPROVED** [review]

**Pull request merged**